packages/cli/commands/accounts/list.js

```diff
@@ -97,7 +102,9 @@ exports.handler = async options => {
 
   logger.log(i18n(`${i18nKey}.configPath`, { configPath }));
   logger.log(
-    i18n(`${i18nKey}.defaultAccount`, { account: config.defaultPortal })
+    i18n(`${i18nKey}.defaultAccount`, {
+      account: config.defaultPortal || config.defaultAccount,
```

Comment:
and a `getDefaultAccount`!

Reply:
Addressed in https://github.com/HubSpot/hubspot-cli/pull/1177/commits/a4d1c22ea6ac6b4368f6c9339bab7ae00bbebb72